src/utils/dateFormatter.ts

```diff
@@ -47,7 +47,7 @@ export const getDateTimeDisplayValue = (timestamp: number): string => {
 };
 
 /**
- * Returns <Date>–<Date> (eg. 7/1-7/15)
+ * Returns <ISO Date> – <ISO Date> (eg. 2023/07/01 - 2023/07/15)
```

Comment:
I'm seeing MM/DD/YYYY – MM/DD/YYYY in Terraware web right now

Reply:
Sorry this is reactive to locale currently. I had pushed a fix to remove locale dependencies entirely.